srcpkgs/mimalloc/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'mimalloc'
+pkgname=mimalloc
+version=1.7.2
+revision=1
+build_style=cmake
+configure_args="-DMI_INSTALL_TOPLEVEL=ON"
```

Comment:
why not add ` -DMI_SECURE=on`? This adds security mitigations

Reply:
@travankor I've thought about adding this flag, but then looked at the performance graphs and saw a performance hit, and thus didn't add the flag.
Though for most benchmarks mimalloc is still faster than glibc's allocator even with mitigations. And Void cares about security hardening, so I've reconsidered my decision and will enable mitigations.